src/test/java/com/tryCloud/runner/CukesRunner.java

```diff
@@ -13,7 +13,7 @@
         features = "src/test/resources/features",
         glue = "com/tryCloud/step_defs",
         dryRun = false,
-        tags = "@wip"
+        tags = "@wipt"
```

Comment:
Hey Than your code is good just delete your tag @wipt cause it cause problem on master  branch, so just delete tag and its good for approval :)